controlplane/src/core/bufservices/PlatformService.ts

```diff
@@ -8207,6 +8317,85 @@ export default function (opts: RouterOptions): Partial<ServiceImpl<typeof Platfo
       });
     },
 
+    enableGraphPruningForTheNamespace: (req, ctx) => {
```

Comment:
In the future, we should think about a generic method instead of creating a RPC method for every new feature.

Reply:
Created a ticket for it